keyboards/honeycomb/honeycomb.h

```diff
@@ -0,0 +1,40 @@
+#ifndef honeycomb_H
+#define honeycomb_H
+
+#include "quantum.h"
+#include "matrix.h"
+#include "backlight.h"
+#include <stddef.h>
+
+#define red_led_off   PORTF |= (1<<6)
+#define red_led_on    PORTF &= ~(1<<6)
```

Comment:
```suggestion
#define red_led_on()    writePinHigh(F6)
```

Reply:
This should use `writePinLow` instead of `writePinHigh`.
```suggestion
#define RED_LED_ON()    writePinLow(F6)
```